packages/google_maps_flutter/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -276,6 +279,26 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
           }
           break;
         }
+      case "map#takeSnapshot":
+        {
+          if (googleMap != null) {
```

Comment:
nits:
Style wise, we prefer to bail early. Do you mind to change it to something like:
```
if (googleMap == null) {
  result.error("GoogleMap uninitialized", "takeScreenshot", null);
  return;
}
...
```

Reply:
Not really. It matches other cases like "map#getLatLng" or "map#getVisibleRegion".

Reply:
sg